nservicebus/messaging/uniformsession.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Uniform Session between message session and pipeline
+summary: Uniform Session allows to bridge the gap between the message session and the pipeline context and enables dependency injection.
+reviewed: 2017-10-27
+component: UniformSession
+---
+
+NServiceBus v6 and higher introduced a few design changes that how code that is using previous versions is structured. The [changes](/nservicebus/upgrades/5to6/moving-away-from-ibus.md) are:
+
+- In previous versions, the `IBus` interface was automatically registered in the IOC container. In Version 6, the new context-aware interfaces, for example, `IEndpointInstance`, `IMessageSession` and `IMessageHandlerContext`, etc., are not automatically registered in dependency injection.
+- In Versions 5 and below, when a custom component was registered in the container, the custom component had access to the `IBus` instance via dependency injection.
```

Comment:
the second point seems to be closely related to point one (DI), should those really be split?

Reply:
Good to unify